docs/sources/flow/getting-started/collect-opentelemetry-data.md

```diff
@@ -0,0 +1,144 @@
+
+Before components can receive OpenTelemetry data, you must have a component
+responsible for exporting the OpenTelemetry data somewhere. An OpenTelemetry
+_exporter component_ is responsible for writing (that is, exporting)
+OpenTelemetry data to an external system.
+
+In this task, we will use the [otelcol.exporter.otlp][] component to send
+OpenTelemetry data to a server using the OpenTelemetry Protocol (OTLP). Once an
+exporter component is defined, other Grafana Agent Flow components can be used
+to forward data to it.
+
+> Refer to the list of available [Components][] for the full list of
+> `otelcol.exporter` components that can be used to export OpenTelemetry data.
+>
+> [Components]: {{< relref "../reference/components/" >}}
+
+To configure a `otelcol.exporter.otlp` component for exporting OpenTelemetry
+data, complete the following steps:
+
+1. Add the following `otelcol.exporter.otlp` component to your configuration
+   file:
+
+   ```river
+   otelcol.exporter.otlp "LABEL" {
+     client {
+       url = "HOST:PORT"
+     }
+   }
+   ```
+
+2. Replace `LABEL` with a label to use for the component, such as `default`.
+   The label chosen must be unique across all `otelcol.exporter.otlp`
+   components in the same configuration file.
+
+3. Replace `HOST` with the hostname or IP address of the server to send
+   OpenTelemetry data to.
+
+4. Replace `PORT` with the port of the server to send OpenTelemetry data to.
+
+5. If your server requires basic authentication, complete the following:
+
+    1. Add the following `otelcol.auth.basic` component to your configuration file:
+
+       ```river
+       otelcol.auth.basic "BASIC_AUTH_LABEL" {
+         username = "USERNAME"
+         password = "PASSWORD"
+       }
+       ```
+
+    2. Replace `BASIC_AUTH_LABEL` with a label to use for the component, such
+       as `default`. The label chosen must be unique across all
+       `otelcol.auth.basic` components in the same configuration file.
+
+    3. Replace `USERNAME` with the basic authentication username to use.
+
+    4. Replace `PASSWORD` with the basic authentication password or API key to
+       use.
+
+    5. Add the following line inside of the `client` block of your
+       `otelcol.exporter.otlp` component:
+
+       ```
+       auth = otelcol.auth.basic.BASIC_AUTH_LABEL.handler
+       ```
+
+    6. Replace `BASIC_AUTH_LABEL` with the label used for the
+       `otelcol.auth.basic` component in step 2.
+
+6. If you have more than one server to export metrics to, create a new
+   `otelcol.exporter.otlp` component for each additional server.
+
+> `otelcol.exporter.otlp` sends data using OTLP over gRPC (HTTP/2). To send to
+> a server using HTTP/1.1, follow the steps above but use the
+> [otelcol.exporter.otlphttp component][otelcol.exporter.otlphttp] instead.
+
+The following example demonstrates configuring `otelcol.exporter.otlp` with
+authentication and a component which forwards data to it:
```

Comment:
Let's also mention the use of env vars in passing? Not sure if it's going to be apparent for a first-time user

Reply:
Great point, I would like to see best practices to manage secrets in Flow configurations. I guess env vars based mgmt of secrets would be the first step.

Reply:
> Let's also mention the use of the env() function in passing? Not sure if it's going to be apparent for a first-time user

I added a comment describing what it's doing in the config. Thanks!